java/src/main/java/api/example/java/controllers/BaseController.java

```diff
@@ -0,0 +1,54 @@
+package api.example.java.controllers;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+
+import api.example.java.Config;
+import api.example.java.model.TokenResponse;
+
+public class BaseController {
+
+	@Autowired
+	protected Config config;
+
+	public BaseController() {
```

Comment:
Do we need to explicitly include this default constructor ?

Reply:
auto gen removed.